packages/manager/apps/telecom-spare/package.json

```diff
@@ -0,0 +1,56 @@
+{
+  "name": "@ovh-ux/manager-telecom-spare-app",
+  "version": "1.0.0",
+  "private": true,
+  "scripts": {
+    "build": "webpack",
+    "dev": "webpack-dev-server",
+    "dev:watch": "yarn run dev",
+    "start": "lerna exec --stream --scope='@ovh-ux/manager-telecom-spare-app' --include-filtered-dependencies -- yarn run build",
+    "start:dev": "lerna exec --stream --scope='@ovh-ux/manager-telecom-spare-app' --include-filtered-dependencies -- yarn run dev",
+    "start:watch": "lerna exec --stream --parallel --scope='@ovh-ux/manager-telecom-spare-app' --include-filtered-dependencies -- yarn run dev:watch"
+  },
+  "dependencies": {
```

Comment:
i looks like you have many unused dependencies (example: leaflet), please remove the one that a re not used

Reply:
I need leaflet to display map to order modem and phones.